woocommerce/woocommerce
# Create workflow for uploading zip file to every asset

This PR adds:

1. Migration of all unit tests to GitHub actions.
2. Adds a workflow which generate a zip file.

To test:
1. Make sure all unit tests pass.
2. Travis build is still being run (we won't remove it yet till the migration is completed for e2e tests as well).
3. Check that zip file is generated for every PR in https://github.com/woocommerce/woocommerce/actions.

Towards #28716 (but does not closes the issue as many enhancements are still pending)
